src/pysdmx/model/message.py

```diff
@@ -0,0 +1,36 @@
+"""Message module.
+
+This module contains the enumeration for the different types of messages that
+can be written.
+"""
+
+from datetime import datetime
+from enum import Enum
+from typing import Optional
+import uuid
+
+from msgspec import Struct
+
+
+class ActionType(Enum):
+    """ActionType enumeration.
+
+    Enumeration that withholds the Action type for writing purposes.
+    """
+
+    Append = "append"
+    Replace = "replace"
+    Delete = "delete"
+    Information = "information"
+
+
+class Header(Struct, frozen=True, kw_only=True):
+    """Header for the SDMX messages."""
+
+    id: str = str(uuid.uuid4())
+    test: bool = True
+    prepared: datetime = datetime.now()
```

Comment:
Thanks!. As mentioned, I would avoid "naive" datetime objects by specifying a default time zone of UTC. As mentioned in the Python docs for the datetime module: "Warning: Because naive datetime objects are treated by many datetime methods as local times, it is preferred to use aware datetimes to represent times in UTC. As such, the recommended way to create an object representing the current time in UTC is by calling datetime.now(timezone.utc)"

The rest looks good to me, thanks a lot :)

Reply:
Okay, adding it, every day you learn something new 😄

Reply:
And this never stops, in IT 😂